tests/pr_test.go

```diff
@@ -10,22 +10,23 @@ import (
 
 // Use existing resource group
 const resourceGroup = "geretain-test-resources"
-const completeExampleDir = "examples/complete"
+const basicExampleDir = "examples/basic"
 
 func setupOptions(t *testing.T, prefix string, dir string) *testhelper.TestOptions {
 	options := testhelper.TestOptionsDefaultWithVars(&testhelper.TestOptions{
 		Testing:       t,
 		TerraformDir:  dir,
 		Prefix:        prefix,
 		ResourceGroup: resourceGroup,
+		Region:        "us-south", // Locking into us-south as SCC instances can only be created in one of the following regions ["us-south", "ca-tor", "eu-es", "eu-de", "eu-fr2"]
```

Comment:
Instead of locking into us-south and only ever testing this region, can you add a new file yaml here (call it `cloudinfo-region-scc-prefs.yaml`) -> https://github.com/terraform-ibm-modules/common-dev-assets/blob/main/common-go-assets with the supported regions, and update the test to use:

```
BestRegionYAMLPath:         "../common-dev-assets/common-go-assets/cloudinfo-region-scc-prefs.yaml",
```